e2e/wdio/platform/pages/action-bar.po.ts

```diff
@@ -0,0 +1,20 @@
+import { BaseComponentPo } from './base-component.po';
+import { waitForElDisplayed, waitForPresent } from '../../driver/wdio';
+
+export class ActionBarPo extends BaseComponentPo {
+    url = '/action-bar';
+    root = '#page-content';
+
+    pageTitle = 'app-action-bar-header h1';
+    actionBtnArr = 'fdp-button button';
+    menuItems = 'div fdp-menu-item';
+    backBtnArr = 'fdp-action-bar button[title=\'Go Back\']';
```

Comment:
try use 'fdp-action-bar .fd-action-bar__back button' to aviod hardcode Go Back

Reply:
done